frontend/scss/_importsCore.scss

```diff
@@ -131,7 +131,7 @@
 @import 'pages/p-artwork-show';
 @import 'pages/p-issue-show';
 @import 'pages/p-issuearticle-show';
-@import 'pages/p-page-show';
+@import 'pages/p-landingpage-show';
```

Comment:
At this point both of these files exist and are needed. Don't replace, just add the landingpage-show import.